tests/test_requirements/additional_requirements.txt

```diff
@@ -0,0 +1,5 @@
+torch
```

Comment:
thoughts on renaming this to tutorials_requirements.txt (or just removing it for now since we don't use tutorials tests right now), and taking out runhouse[sagemaker] + runhouse[aws]? for those we can just directly specify in the command rather than including runhouse libraries in reqs.txt